src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,130 @@
+.main {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: flex-start;
+  margin: -4px auto;

```

Comment:
Using a negative margin it's not a good practice, better to set the less margin to the previous and next elements.